pcdscalc/be_lens_calcs.py

```diff
@@ -201,17 +193,17 @@ def get_lens_set(set_number_top_to_bot, filename=None):
     if os.stat(filename).st_size == 0:
         logger.error('The file is empyt: %s', filename)
         return
-    with open(filename, 'rb') as lens_file:
-        sets = np.load(lens_file, allow_pickle=True)
-        if set_number_top_to_bot not in range(1, sets.shape[0]):
+    with open(filename) as lens_file:
+        sets = json.loads(lens_file.read())
+        if set_number_top_to_bot not in range(1, len(sets)):
             logger.error('Provided and invalid set_number_top_to_bottom %s, '
                          'please provide a number from 1 to %s ',
-                         set_number_top_to_bot, sets.shape[0])
+                         set_number_top_to_bot, len(sets))
             return
     return sets[set_number_top_to_bot - 1]
```

Comment:
Just found a little error here, I assumed we'll have more than one set here, so if we have something like `[2, 0.3, 2, 0.4]` instead of `[[set1], [set2]]` it will not work properly - fixing this right now

Reply:
ok fixed this in 8a38673